IntegrationTests/UITests/CommandsDialogs/FormBrowseTests.LeftPanel.Remotes.cs

```diff
@@ -183,12 +177,12 @@ private TreeNode GetRemoteNode(FormBrowse form)
             return remotesNode;
         }
 
-        private void RunFormTest(Action<FormBrowse> testDriver)
+        private void RunFormTest(Action<TreeNode> testDriver)
```

Comment:
💡 Consider renaming this test helper since it no longer operates with the same pattern as other `Form` tests.